OPERATING.md

```diff
@@ -192,15 +192,48 @@ An example daisy chaining environment is provided in [/test-suite/daisy-chaining
 * Azure Service Bus may require some parameters as part of the URL configuration. For example: `PS_AMQP_BROKER=amqps://<NAME>.servicebus.windows.net/;SharedAccessKeyName=<KEY NAME>;SharedAccessKey=<KEY VALUE>`
 
 ## Object storage
-Data stored:
-    EhrExtract attachments of MHS Inbound, pre-signed S3 url is generated for stored attachments      
-Filename convention:
-    Attachment files are named as {conversationId}_{documentId} where documentId is the name of the file which includes an extension.
-    ConversationId - Task conversation ID
-Configuration:
-    The app uses a number of attempts to upload attachments. It is configured in retry policy. 
-    Generated stored attachments will be available for 60 min to be downloaded, after this time limit the download link will be invalidated
-    although no files will be deleted from S3 bucket.
+
+GP2GP messaging splits the patient's Electronic Health Record (EHR) into an EHR Extract and associated attachments. 
+The adaptor uses AWS / Azure object storage to manage the attachments.
+
+### Data stored
+It is the responsibility of the GP System Supplier to manage the data stored in object storage post transfer. 
+
+#### Incomplete transfers
+The sending GP2GP system can split attachments into multiple parts that the adaptor must reassemble. It could also send 
+the EHR Extract itself as a compressed attachment. Therefore, an incomplete / failed transfer could have both of these 
+uploaded to object storage. If a transfer fails they will not be deleted by the adaptor. 
+
+##### Complete transfers
+Assembled attachments will be uploaded to object storage. The adaptor obtains a URL for each attachment, which it inserts
+into the returned FHIR bundle. When using AWS S3 this URL is pre-signed and valid for 60 minutes. After this time, the 
+download link will be invalidated, although no files will be deleted from the S3 bucket.   
+
+The pre-assembled attachment parts are removed from storage when an attachment is assembled. However, if the transfer 
+contains a compressed EHR Extract this is not removed from storage automatically.
+
+### Filename convention
+Attachment files are named as `conversationId_documentId` where `documentId` is the name of the file (including extension)
+and `conversationId` is an identifier unique to transfer. 
```

Comment:
😢 Shame there is no way for an NME to get that `conversationId` for a patient.

Reply:
Yes that could be useful. Although, it's currently the responsibility to the NME to keep track of the conversation IDs, as its either passed in or generated by the façade and returned in the response header.